SConstruct

```diff
@@ -249,6 +249,11 @@ o.Add(
 	"/usr/lib",
 )
 
+o.Add(
+	"PYTHONPATH",
```

Comment:
A bit funny that it's named generically but only used for tests. Don't we add `TEST_` to other variables like that. Are you thinking of using it outside the tests as well?

Reply:
I wasn't sure which way to go, but it seemed plausible that we'd use it for other things, and I thought having a matching set of CPPPATH, LIBPATH and PYTHONPATH made sense. Not particularly attached to it though if you feel strongly about it.